app/models/discourse_livestream/topic_chat_channel.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module DiscourseLivestream
+  class TopicChatChannel < ActiveRecord::Base
+    self.table_name = "topic_chat_channels"
```

Comment:
Is there any reason to prepend with `livestream` or something in case core ever needs a table named the same...? 🤷

Reply:
Updated in https://github.com/discourse/discourse-livestream/pull/21/commits/b8d671f7f1b6bfd51beaa443cea5f6bfecd10625